install-dependencies/README.md

```diff
@@ -0,0 +1,76 @@
+# GitHub Action Install Dependencies
+
+<!-- prettier-ignore-start -->
+<!-- action-docs-description source="action.yaml" -->
+## Description
+
+GitHub Action that installs dependencies in a Node based repository
+<!-- action-docs-description source="action.yaml" -->
+<!-- prettier-ignore-end -->
+
+## Usage
+
+```yaml
+jobs:
+  build:
+    steps:
+      - name: Install Dependencies
+        uses: open-turo/actions-node/install-dependencies@v5
+        with:
+          ## example value for github-token provided below
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          npm-token: ${{ secrets.NPM_TOKEN }}
+```
+
+You can specify an S3 bucket to cache node_modules in order to speed up dependency installation, in which case you will need to provide an AWS access id and secret key.
+
+```yaml
+jobs:
+  build:
+    steps:
+      - name: Install Dependencies
+        uses: open-turo/actions-node/install-dependencies@v5
+        with:
+          ## example value for github-token provided below
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          s3-bucket-name: <bucket-name>
+          s3-bucket-region: us-east-1
+        env:
+          AWS_ACCESS_ID: ${{ secrets.AWS_ACCESS_ID }}
```

Comment:
I don't think we should do it like this, but instead use something like this before https://github.com/aws-actions/configure-aws-credentials